src/Chapter1.hs

```diff
@@ -554,7 +556,11 @@ value after "=" where the condition is true.
 Casual reminder about adding top-level type signatures for all functions :)
 -}
 
-mid x y z = error "mid: not implemented!"
+mid :: Int -> Int -> Int -> Int
+mid x y z 
+  | ((x > y) && (x < z)) || ((x < y) && (x > z)) = x
+  | ((y > x) && (y < z)) || ((y < x) && (y > z)) = y
```

Comment:
Awesome!
You can also skip the brackets 👍🏼 

```suggestion
  | (x > y && x < z) || (x < y && x > z) = x
  | (y > x && y < z) || (y < x && y > z) = y
```

Note that outside brackets could be removed too due to the precedence (ands will work first by priority), but I think it is fine to leave it for the readability ✍🏼